api/tempo/v1alpha1/tempostack_types.go

```diff
@@ -589,8 +589,21 @@ type TempoGatewaySpec struct {
 	//
 	// +optional
 	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Jaeger gateway Ingress Settings"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Gateway Ingress Settings"
 	Ingress IngressSpec `json:"ingress,omitempty"`
+
+	// RBAC defines RBAC options.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Query RBAC Settings"
+	RBAC RBACSpec `json:"rbac,omitempty"`
+}
+
+// RBACSpec defines RBAC options.
+type RBACSpec struct {
+	// Enabled defines if the query RBAC should be enabled.
```

Comment:
let's add the kubebuilder params (`// +kubebuilder:validation:Optional`, `// +operator-sdk:csv:customresourcedefinitions`, etc.)